### PR TITLE
feat: card component #P23-114

### PR DESCRIPTION
**Pull request by aniasgit**

### CODE REVIEW

File: apps/storybook/stories/Card.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
```
[the remark this answers is not in the document]
Comment (aniasgit) [reply apps/storybook/stories/Card.stories.tsx:1]: You're right, I already changed it

File: packages/ui/Card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { ReactNode } from "react";
+import styled, { css } from "styled-components";
```
Comment (wyola) [inline packages/ui/Card/index.tsx:4]: { css } is not used - you might think about removing it

File: packages/ui/Card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	verticalFit,
+}: CardProps) => {
+	return (
+		<CardWrapperStyled horizontalFit={horizontalFit} verticalFit={verticalFit}>
+			<CardStyled
+				padding={padding}
+				margin={margin}
+				horizontalFit={horizontalFit}
+				verticalFit={verticalFit}>
+				<CardContenetStyled>{children}</CardContenetStyled>
```
Comment (wyola) [inline packages/ui/Card/index.tsx:28]: typo - "Contenet"

**Review by wyola — COMMENTED** [review]
I think that this component could be just a single `div` with padding, so that content does not touch the borders. Padding changes on different screen sizes, so there could be a media query in the styling.

**Review by bdeluga — COMMENTED** [review]
unused css from styled-components and small typo, fix quickly and we can approve 🚀

File: packages/ui/Card/index.tsx (hunk 2 above)
Comment (aniasgit) [reply packages/ui/Card/index.tsx]: done

File: packages/ui/Card/index.tsx (hunk 3 above)
Comment (aniasgit) [reply packages/ui/Card/index.tsx]: In the latest changes, I removed this part.

**Comment by AzulloCyll** [comment]
In figma there is 1px border on Card component when it's on white background but there is no border when Card is on dark. 
I suggest adding border prop. Maybe with default value.

**Review by AzulloCyll — APPROVED** [review]
Looks good form me

**Review by zuzannapaw — APPROVED** [review]

**Pull request merged**